lib/rules/shorthand-property-no-redundant-values/__tests__/index.mjs

```diff
@@ -121,286 +121,286 @@ testRule({
 			fixed: 'a { overflow: scroll; }',
 			message: messages.rejected('scroll scroll', 'scroll'),
 			line: 1,
-			column: 5,
+			column: 15,
 			endLine: 1,
-			endColumn: 29,
+			endColumn: 28,
 		},
 		{
 			code: 'a { gap: 1rem 1rem; }',
 			fixed: 'a { gap: 1rem; }',
 			message: messages.rejected('1rem 1rem', '1rem'),
 			line: 1,
-			column: 5,
+			column: 10,
 			endLine: 1,
-			endColumn: 20,
+			endColumn: 19,
 		},
 		{
 			code: 'a { margin: 1px 1px; }',
 			fixed: 'a { margin: 1px; }',
 			message: messages.rejected('1px 1px', '1px'),
 			line: 1,
-			column: 5,
+			column: 13,
 			endLine: 1,
-			endColumn: 21,
+			endColumn: 20,
 		},
 		{
 			code: 'a { margin: 1Px 1pX; }',
 			fixed: 'a { margin: 1Px; }',
 			message: messages.rejected('1Px 1pX', '1Px'),
 			line: 1,
-			column: 5,
+			column: 13,
```

Comment:
[suggestion] Maybe include the end line and end column in each test to have more coverage?

Reply:
Ill add `endLine`/`endColumn` to the `!important` test cases.